File: tests/cases/str_enum.py
from enum import Enum


class Colors(str, Enum):
    RED = "red"
    GREEN = "green"
    BLUE = "blue"


def show():
    color_map = {Colors.RED: "1", Colors.GREEN: "2", Colors.BLUE: "3"}
    a = Colors.GREEN
    if a == Colors.GREEN:
        print("green")
    else:
        print("Not green")
    print(len(color_map))


if __name__ == "__main__":
    show()

File: pyjl/tests/tests_py2many-py2many/str_enum.jl
abstract type AbstractColors end

struct Colors::str, Enum, AbstractColors 
end

function show()
color_map = Dict(Colors.RED => "1", Colors.GREEN => "2", Colors.BLUE => "3")
a = Colors.GREEN
if a == Colors.GREEN
println("green");
else
println("Not green");
end
println(length(color_map));
end

function main()
show();
end

main()
